### PR TITLE
a couple suggestions

### PR DESCRIPTION
**Pull request by greglandrum**
This adds a couple of new tests and changes the name of the new function from `copyRelativeStereoGroups()` to `copyEnhancedStereoGroups()`.

Unfortunately a bit of reformatting got mixed in too...

### CODE REVIEW

File: Code/GraphMol/ChemReactions/catch_tests.cpp [hunk 1]
```diff
@@ -56,43 +56,90 @@ TEST_CASE("Github #1632", "[Reaction,PDB,bug]") {
 // static void clearAtomProps()
 
 TEST_CASE("Github #2366 Enhanced Stereo", "[Reaction,StereoGroup,bug]") {
```
Comment (d-b-w) [inline Code/GraphMol/ChemReactions/catch_tests.cpp:58]: Not sure how we're using Catch2 tags yet. Maybe `bug` isn't a helpful test tag.

File: Code/GraphMol/ChemReactions/catch_tests.cpp [hunk 2]
```diff
@@ -56,43 +56,90 @@ TEST_CASE("Github #1632", "[Reaction,PDB,bug]") {
 // static void clearAtomProps()
```
Comment (d-b-w) [inline Code/GraphMol/ChemReactions/catch_tests.cpp:56]: whoops. I should bring this helper back, clearly (har har).

**Review by d-b-w — APPROVED** [review]
I guess it does feel like we're settling on the "enhanced" terminology. In the first commits, I tried to stick to "relative" or just "stereo group", because I was thinking that I wanted it to be internally meaningful to the RDkit. Using "enhanced" as the name makes it feel really tied to SDF.

OTOH, I suppose that people who are asking for this feature are going to be familiar with it from SD files.

File: Code/GraphMol/ChemReactions/catch_tests.cpp (hunk 1 above)
Comment (greglandrum) [reply Code/GraphMol/ChemReactions/catch_tests.cpp:58]: agreed.

**Comment by greglandrum** [comment]
Re: terminology
Yeah, I think "enhanced stereo" is as good of a term as any for these things. It's immediately clear for people who know about the MDL version yet still descriptive for others.

**Pull request merged**